### PR TITLE
feat(respondr): Adding class guest to body for css

### PR DESCRIPTION
**Pull request by jgribonvald**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [ ] tests are included
-   [ ] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [ ] view conforms with [WCAG 2.0 AA][]

##### Description of change

Adding a css class for guest page, to be able to customize the skin depending on authenticated or not authenticated, without need to deploy a new skin only for this need.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl [hunk 1]
```diff
@@ -656,6 +656,7 @@
           <xsl:attribute name="class">
             <xsl:text>up dashboard portal fl-theme-mist</xsl:text>
             <xsl:if test="$PORTAL_VIEW='focused'"> up-focused</xsl:if>
+            <xsl:if test="$AUTHENTICATED!='true'"> guest</xsl:if>
```
Comment (ChristianMurphy) [inline uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl]: Where possible use the `up` prefix for custom css.
`guest` -> `up-guest`

Comment (jgribonvald) [reply uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl]: done !

File: uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl [hunk 2]
```diff
@@ -798,7 +799,11 @@
             <xsl:copy-of select="."/>
         </xsl:for-each>
     </head>
-    <body class="up dashboard portal fl-theme-mist detachedHeader">
+    <body>
+        <xsl:attribute name="class">
+            <xsl:text>up dashboard portal fl-theme-mist detachedHeader</xsl:text>
```
Comment (drewwills) [inline uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl]: Is `fl-theme-mist` needed at all any longer?

Comment (jgribonvald) [reply uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl]: Not sure for me !

Comment (drewwills) [reply uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl]: There was another PR earlier to remove all Fluid classes, and I think that's still something we want to do.  (The reason we didn't merge it is that many of them were used in jQuery selectors and in custom style rules... so we need to clean that stuff up.)

Comment (ChristianMurphy) [reply uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl]: #1051 was the PR

**Pull request merged**